aten/src/ATen/native/cpu/GridSamplerKernel.cpp

```diff
@@ -627,7 +635,9 @@ struct ApplyGridSample<scalar_t, 2, GridSamplerInterpolation::Nearest, padding>
     integer_t gInp_offset_arr[iVec::size];
     i_gInp_offset.store(gInp_offset_arr);
 
-    #pragma unroll
+    #if (!defined _MSC_VER)  
```

Comment:
Wow! I didn't know you could write defined declarations this way.

Reply:
I can change it to the other convention if you want (and haven't done so already).

Reply:
Yes please!

Reply:
Updated to use the standard convention.